pvfactors/irradiance/models.py

```diff
@@ -23,19 +23,27 @@ class IsotropicOrdered(BaseModel):
     cats = ['ground', 'front_pvrow', 'back_pvrow']
     irradiance_comp = ['direct']
 
-    def __init__(self):
-        """Initialize irradiance model values that will be saved later on."""
+    def __init__(self, rho_front=0.01, rho_back=0.03):
+        """Initialize irradiance model values that will be saved later on.
+
+        Parameters
+        ----------
+        rho_front : float, optional
+            Reflectivity of the front side of the PV rows (default = 0.01)
+        rho_back : float, optional
+            Reflectivity of the back side of the PV rows (default = 0.03)
+        """
         self.direct = dict.fromkeys(self.cats)
         self.total_perez = dict.fromkeys(self.cats)
         self.isotropic_luminance = None
-        self.rho_front = None
-        self.rho_back = None
+        self.rho_front = rho_front
+        self.rho_back = rho_back
```

Comment:
now reflectivity values are passed to the irradiance model at initialization, not in the ``fit()`` function; which makes more sense since they are always floats